sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PrivateFieldUsedAsLocalVariable.CSharp9.cs

```diff
@@ -0,0 +1,28 @@
+﻿record Record
+{
+    private int F0 = 0; // Compliant - unused
+    private int F1 = 0; // Compliant - FN
+    public int F2 = 0; // Compliant - Public
```

Comment:
also add an `internal`, `protected`, `protected internal` and `private protected`?

Reply:
I've assumed they are covered in the test cases we had already but they are not (not all). I've added them.